hidimstat/cpi.py

```diff
@@ -42,7 +44,7 @@ def __init__(
         imputation_model,
         n_permutations: int = 50,
         loss: callable = root_mean_squared_error,
-        score_proba: bool = False,
+        method: str = "predict",
```

Comment:
You need to update the docstring above, as well as for LOCO and PI.

Reply:
done